pkg/kubelet/dockershim/docker_checkpoint.go

```diff
@@ -82,19 +85,24 @@ func NewPodSandboxCheckpoint(namespace, name string, data *CheckpointData) Docke
 	}
 }
 
+// MarshalCheckpoint encodes the PodSandboxCheckpoint instance to a json object
 func (cp *PodSandboxCheckpoint) MarshalCheckpoint() ([]byte, error) {
 	cp.Checksum = checksum.New(*cp.Data)
 	return json.Marshal(*cp)
 }
 
+// UnmarshalCheckpoint decodes the blob data to the PodSandboxCheckpoint instance
 func (cp *PodSandboxCheckpoint) UnmarshalCheckpoint(blob []byte) error {
 	return json.Unmarshal(blob, cp)
 }
 
+// VerifyChecksum verifies wheather the PodSandboxCheckpoint's data checksum is
```

Comment:
wheather -> whether

Reply:
Thank you for you comment， I will be careful for my spelling.